server/src/main/java/org/spine3/server/aggregate/AggregateRepository.java

```diff
@@ -355,12 +355,15 @@ public void store(A aggregate) {
         return result;
     }
 
-    private static <I> IllegalStateException unableToLoadEvents(I id) {
-        final String errMsg = format(
-                "Unable to load events for the aggregate with ID: %s",
-                idToString(id)
-        );
-        throw new IllegalStateException(errMsg);
+    private static void checkAggregateStateRecord(AggregateStateRecord aggregateStateRecord) {
```

Comment:
`AggregateRepositoryShould#reject_invalid_AggregateStateRecord_instance()` tests it.